zgnroot/include/zgnr/gl-base-technique.h

```diff
@@ -20,6 +33,10 @@ struct zgnr_gl_base_technique {
   struct {
     struct zgnr_gl_vertex_array *vertex_array;  // nullable
     bool vertex_array_changed;
+
+    enum zgnr_gl_base_technique_draw_method draw_method;
+    union zgnr_gl_base_technique_draw_args args;
+    bool draw_info_changed;
```

Comment:
```suggestion
    bool draw_method_changed;
```

"draw info" seems a bit too broad in meaning. This could include vertex array, texture, etc.

Reply:
> "draw info" seems a bit too broad in meaning

Yes, I agree this. but I think when call `draw_array` twice, but draw method isn't changed.
So I named this `draw_info` instead of `draw_method` to inform that this monitors both method and args.